dask_expr/concat.py

```diff
@@ -0,0 +1,121 @@
+import functools
+
+import pandas as pd
+from dask.dataframe import methods
+from dask.dataframe.dispatch import make_meta, meta_nonempty
+from dask.dataframe.utils import check_meta, strip_unknown_categories
+from dask.utils import apply, is_dataframe_like, is_series_like
+
+from dask_expr.expr import AsType, Expr
+
+
+class Concat(Expr):
+    _parameters = ["join", "ignore_order", "_kwargs"]
+    _defaults = {"join": "outer", "ignore_order": False, "_kwargs": {}}
+
+    @property
+    def _frame(self):
+        return self.operands[len(self._parameters) :]
```

Comment:
Pluralize into `_frames`?

You could also use `.dependencies()` here instead, which is just all `Expr`s in operands.  Maybe that's cleaner.